Lib/test/test_unittest/support.py

```diff
@@ -1,7 +1,7 @@
 import unittest
 
 
-class TestEquality(object):
+class TestEquality:
```

Comment:
I hate to be _that guy_, but could we do changes like this and the `super()` changes in another PR? This makes the diff much bigger and harder to review, and it will make it harder to spelunk through `git blame` in the future to see which files had to change to make colourised `unittest` possible

Reply:
Sure, removed!